code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -98,7 +134,107 @@
 		qdel(O)
 	else
 		return ..()
-		
+
+/mob/living/simple_animal/mouse/attack_ghost(mob/dead/observer/user)
+	user.possess_mouse(src)
+
+/mob/living/simple_animal/mouse/start_pulling(atom/movable/AM, state, force, supress_message)
+	return FALSE
+
+/mob/living/simple_animal/mouse/CtrlClickOn(atom/A)
+	face_atom(A)
+	if(!isturf(loc)) return
+	if(next_move > world.time) return
+	if(!A.Adjacent(src)) return
+
+	if(!can_eat(A))
+		return FALSE
+	
+	eating = TRUE
+	layer = MOB_LAYER
+	visible_message("<span class='danger'>[src] starts eating away [A]...</span>",
+						 "<span class='notice'>You start eating the [A]...</span>")
+	if(do_after(src, 30, FALSE, A))
+		if(QDELETED(A))
+			return
+		visible_message("<span class='danger'>[src] finishes eating up [A]!</span>",
+						 "<span class='notice'>You finish up eating [A]</span>")
+		A.mouse_eat(src)
+		GLOB.mouse_food_eaten++
+
+	eating = FALSE
+	layer = BELOW_OPEN_DOOR_LAYER
+
+/mob/living/simple_animal/mouse/proc/can_eat(atom/A)
+	. = FALSE
+
+	if(eating)
+		return FALSE
+	if(is_type_in_list(A, GLOB.mouse_combestible))
+		return TRUE
+	if(istype(A, /obj/item/reagent_containers/food) && !(locate(/obj/structure/table) in get_turf(A)))
```

Comment:
Is it intentional that mouse_combestible bypasses table checks? also combestible isnt even a real word

Reply:
Yes it is, fixed combestible